core/linux-aarch64/PKGBUILD

```diff
@@ -41,16 +41,15 @@ prepare() {
   cd $_srcname
 
   echo "Setting version..."
-  scripts/setlocalversion --save-scmversion
   echo "-$pkgrel" > localversion.10-pkgrel
   echo "${pkgbase#linux}" > localversion.20-pkgname
 
   # add upstream patch
-  git apply --whitespace=nowarn ../patch-${pkgver}
+  patch -Np1 < ../patch-${pkgver}
```

Comment:
`git apply` didn't do anything here for some reason

Reply:
it doesn't work if you build linux-aarch64 inside of PKGBUILDs git tree

Reply:
I see, so the package maintainers are just building using an external `SRCDEST` by default?